PF-session2/pf2_assignment1/Pyramid.java

```diff
@@ -0,0 +1,116 @@
+
+/**
+ * This class has three methods to generate spaces,numbers and the final pattern.
+ *    1
+ *   121
+ *  12321
+ * 1234321
+ *123454321
+ * 1234321
```

Comment:
Class level comment should go just above the class, and not before the package declaration or imports.